src/pages/TaskDetails/TaskDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import { CardContent, Typography } from '@mui/material'
+import Card from '@mui/material/Card'
+
+// de la url un id, y ese de id muestra sus detalles;
+// una tarea 3 atributos:  id, name, description
+
+const task = {
+  id: 686,
+  name: 'First Task',
+  description: 'This is the first task',
+}
+
+export const TaskDetails: React.FC = () => {
```

Comment:
Don't need React.FC as is just a function that we will use later as a component, it will be infered